Dockerfile

```diff
@@ -7,11 +7,11 @@
 # where <path/to/WARCs> is a file path that only contains WARC files and directories.
 
 # When the container is running, run the following commands to start Solr and Tomcat:
-# export SOLRWAYBACK_VERSION=4.4.2
-# export APACHE_TOMCAT_VERSION=8.5.60
-# export SOLR_VERSION=7.7.3
-# ./unpacked-bundle/solrwayback_package_$SOLRWAYBACK_VERSION/solr-$SOLR_VERSION/bin/solr start
-# ./unpacked-bundle/solrwayback_package_$SOLRWAYBACK_VERSION/apache-tomcat-$APACHE_TOMCAT_VERSION/bin/startup.sh
+# export SOLRWAYBACK_VERSION 5.1.0
+# export APACHE_TOMCAT_VERSION 9
+# export SOLR_VERSION 9
```

Comment:
Sorry. these are bash variables, they need a =-sign in between variable and value withoug spaces to work.

Reply:
Indeed! Done